src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -133,14 +133,14 @@ const BioAndDMSetting = ({
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: colors.darkBlue,
+    backgroundColor: COLORS.blue,
     borderRadius: 15,
     paddingHorizontal: 12,
     marginTop: 20
   },
-  editPromptLabel: {color: colors.blueSea10, textDecorationLine: 'underline'},
+  editPromptLabel: {color: COLORS.blueSea, textDecorationLine: 'underline'},
   bioText: {
-    color: '#F5F5F5',
+    color: COLORS.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values in the style properties have been updated to use the new `COLORS` constant. This change ensures consistency across the application and makes it easier to manage color schemes.
```diff
- backgroundColor: colors.darkBlue,
- editPromptLabel: {color: colors.blueSea10, textDecorationLine: 'underline'},
- bioText: {color: '#F5F5F5',
+ backgroundColor: COLORS.blue,
+ editPromptLabel: {color: COLORS.blueSea, textDecorationLine: 'underline'},
+ bioText: {color: COLORS.lightgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->